### ISSUE TITLE
curseforge remuneration

### ISSUE THREAD

**Issue by kreezxil** [TheEpicBlock/PolyMc#98]
Consider the following:
**CurseForge Rewards Program**
https://support.curseforge.com/en/support/solutions/articles/9000197709-how-to-join-and-redeem-rewards
https://support.curseforge.com/en/support/solutions/articles/9000197898-rewards-program-terms-of-service
https://support.curseforge.com/en/support/solutions/articles/9000197902-reward-program-faq

Then consider that this server ware kills the individual download credit that would have been made a client pack been made and distributed. How does this server ware remedy that problem? Does it download the mods in the background via the cf API for each new client that joins?

**Comment by TheEpicBlock** [comment]
This is quite an interesting topic actually.

I'd love to help give authors more credit for their mods. I just don't believe there's a good solution here. No matter
But from the ethical side, I do definitely agree that mod authors deserve recognition for their mods. I just don't see an easy way for me to help with this. I doubt downloading the mods in the background will help much, the faq seems vague on what exactly counts towards points. I think downloading them repeatedly from the same server would get flagged by CF either way. From a purely technical standpoint, mods don't include a link to their CF page, if they even have one. Having some

**Comment by TheEpicBlock** [comment]
I'd love to help give authors more credit for their mods. I just don't believe there's a good solution to this on my side. I doubt downloading the mods would help much. The faq is vague on what exactly counts towards points and cf will probably flag it as spam either way. Besides, mods don't include links to their curseforge page in a standardized way.

**Comment by Patbox** [comment]
That's server side modding for you. Ideally servers should have a mod adding /mods command like this one https://modrinth.com/mod/mods-command. But yeah, thats normal for server side mods to get less downloads, but there is also big chance that players wouldn't play them otherwise, so it doesn't make them lose anything

**Comment by kreezxil** [comment]
You didn't read what this server does. It says it takes mods that are meant
for universal, knocks out the client-side part, extracts the textures,
sends those as a forced resource pack to the client, and runs the rest of
the mod server side. Literally killing mods as they were meant to be.

If I didn't understand something, please enlighten me.

**Comment by Patbox** [comment]
It's not `killing` mods, it just makes them more accessible to others, as it's easier to join the server than download a modpack. I see it more as way to promote these mods. I also don't see it really impacting downloads that much. Most downloads of (content) mods are from modpacks anyway.

**Comment by TheEpicBlock** [comment]
This is getting out of scope for this issue. PolyMc was made to be a fun project to see how far server-side-only modding can be taken. This might not be how you think mods were meant to be, but it's fun to explore either way. 

As there doesn't seem to be any good way for me to inflate CF download counts to compensate for the existence of this mod, I'm closing this issue. Mod authors definitely deserve credit for what they do, but I'm unsure what PolyMc's place in this is. After all, it is just a tool to get content working server-side.

**Comment by ticticboooom** [comment]
I appreciate this is closed, but I am fuming, Kreezxil.............. there is no need to try and kill innovative ideas, just to keep CF points flooding in. this ise is awesome, could be ana amazing leap forward for m servers. so i suggest you do not use money as a reason to `kill` innovative ideas

**Comment by kreezxil** [comment]
I have a valid reason. Just because you disagree is ok, but you shouldn't get mad about it. I should be allowed to disagree with you.

I recommend the mod author delete my comment and your comment, and to lock this issue.

**Comment by GrokDuckFarmer** [comment]
Let's imagine a movie streaming service, and as all these streaming services do, it pays the movie studios a fee for each time someone streams a movie from them.  Everyone benefits and everyone is happy: the movie studios get money for views of their movies, the streaming service gets money by way of user fees and/or ads, and the end user gets to watch a movie.

Now imagine someone tries to innovate on that.  Innovations is great!  We love innovation!  Yay, innovation!  Said innovator figures out a way to let people stream movies that bypasses the streaming service, and the end users love it because it's so easy for them, but the movie studios don't collect any fees from people watching their movies anymore.  But innovation is great, and we shouldn't let money be a reason to kill off innovation.  That's just dumb.  Full steam ahead on innovation!  Roll forward some years, and people wonder why so few movie studios are making new movies and some are going out of business.  Few people are putting putting together the issue of innovatively bypassing movie fees with the fallout of movie studios not getting paid.

To put it another way, using client mods without compensating the mod authors is akin to downloading a movie without compensating the studio.  Both are theft.  Innovation is great, but innovating ways to steal is not.  Clear?
